cmd/analyzer/analyzer.go

```diff
@@ -348,6 +349,11 @@ func NewService(cfg *config.AnalysisConfig) (*Service, error) {
 			return analyzer.NewAggregateStatsAnalyzer(cfg.Analyzers.AggregateStats, dbClient, logger)
 		})
 	}
+	if cfg.Analyzers.EVMContractsVerifier != nil {
+		analyzers, err = addAnalyzer(analyzers, err, func() (A, error) {
+			return evmverifier.NewEVMVerifierAnalyzer(cfg.Analyzers.EVMContractsVerifier, dbClient, logger)
```

Comment:
WDYT about allowing `cfg.Analyzers.EVMContractsVerifier.ChainName` to be empty, and using the global default when available? Like so:
```go
verifierCfg := cfg.Analyzers.EVMContractsVerifier
if verifierCfg.ChainName == "" && cfg.Source.ChainName != "" {
  # If chain name is not explicitly configured, default to the same chain we're pulling the blocks from.
  verifierCfg.ChainName = cfg.Source.ChainName
}
return evmverifier.NewEVMVerifierAnalyzer(verifierCfg, dbClient, logger)
```

Reply:
Good idea, did this but in `config.go` together with the default value for interval.